src/Components/v2/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface FlickitySettings {
+  draggable?: boolean
+  freeScroll?: boolean
+  wrapAround?: boolean
+  groupCells?: boolean | number | string
+  autoPlay?: boolean | number
+  fullscreen?: boolean
+  fade?: boolean
+  adaptiveHeight?: boolean
+  watchCSS?: boolean
+  hash?: boolean
+  dragThreshold?: number
+  friction?: number
+  selectedAttraction?: number
+  freeScrollFriction?: number
+  imagesLoaded?: boolean
+  lazyLoad?: boolean | number
+  bgLazyLoad?: boolean | number
+  initialIndex?: number
+  accessibility?: boolean
+  setGallerySize?: boolean
+  resize?: boolean
+  cellAlign?: "left" | "center" | "right"
+  contain?: boolean
+  percentPosition?: boolean
+  rightToLeft?: boolean
+  prevNextButtons?: boolean
+  pageDots?: boolean
+}
```

Comment:
Since the lib use is basically a one-off and not a larger framework-like-thing this doesn't need to be in its own file in the root, but rather live inside of the carousel file directly. Devs updating / making changes to the carousel then have a nice list of options immediately available to them, right at the top of the file.

Reply:
Ok I'll move it there.